.factory/automation.yml

```diff
@@ -6,6 +6,12 @@ config:
 build:
   correctness:
 
+    test-docs-content:
+      image: typedb-ubuntu-22.04
+      type: foreground
+      command: |
+        npx antora generate --ui-bundle-url ./minimal-ui-bundle.zip antora-playbook-mock-for-ci.yml
+
```

Comment:
Seems to work fine. Unhappy commit: https://github.com/krishnangovindraj/typedb-docs/commit/e05defed839e751f255c2bff2b76d7da77e00f68